src/string_bytes.h

```diff
@@ -105,6 +105,8 @@ class StringBytes {
 
   static std::string hex_encode(const char* src, size_t slen);
 
+  static size_t keep_buflen_in_range(size_t len);
```

Comment:
It doesn't seem necessary to keep this in the headers. A static method in the .cc file is enough and avoids linking overhead.

Reply:
updated, PTAL @joyeecheung